io/os.go

```diff
@@ -0,0 +1,35 @@
+package io
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+// Copied from https://github.com/corazawaf/coraza/blob/v3/dev/internal/io/file.go
+
+// OSFS implements fs.FS using methods on os to read from the system.
+// Note that this implementation is not a compliant fs.FS, as they should only
+// accept posix-style, relative paths, but as this is an internal implementation
```

Comment:
If this is exposed it's not an internal implementation detail anymore and a public, non-compliant implementation of `fs.FS`. I don't know if it's really a big deal in practice but it is something to think about

Reply:
I would say as long as it works for coraza it is OK to have it.

Reply:
Sounds ok. Let's fix up the comment to read like a public API